app.js

```diff
@@ -15,6 +15,11 @@ app.get('/api/articles',getAllArticles);
 
 app.get('/api/articles/:article_id/comments',getCommentsByArticleId);
 
+app.use((err,req,res,next)=>{
+    
+    res.status(err.status).send({msg:err.msg})
+})
```

Comment:
Unrelated to the ticket, but it would be good to include an error handler for a 500 Internal Server Error!